src/admin.c

```diff
@@ -334,7 +334,7 @@ bool RequestFiles(
         file = "index.html";
     }
 
-    const char *etc_path = ut_locate(BAKE_PROJECT_ID, NULL, UT_LOCATE_ETC);
+    const char *etc_path = ut_locate(ut_getenv("BAKE_PROJECT_ID"), NULL, UT_LOCATE_ETC);
```

Comment:
`BAKE_PROJECT_ID` is defined by bake as a command line parameter

Reply:
Reverted that to macro in the latest commit